pages/batt/A_Sons_Blood.vue

```diff
@@ -0,0 +1,343 @@
+              <li>摧毀班圖草原營地。</li>
+            </ul>
+          </div>
+          <br />
+          <h2>任務提示</h2>
+          <div class="hrn1"></div>
+          <div class="n1">
+            <ol>
+              <li>
+                你的人口上限是150，初始為城堡時代，此關卡可升至<nuxt-link
+                  to="/techs/age/Imperial_Age">帝王時代</nuxt-link>。
+              </li>
+              <li>
+                當你招降葡萄牙人時，不要忽略生產新單位。你需要就緒一旦葡萄牙人都被招降，你就要跟馬木留克奴隸兵人，奧斯曼人和第烏人作戰。
+              </li>
+              <li>
+                摧毀馬木留克奴隸兵港灣大商站是首要任務。他們是馬木留克奴隸兵人國庫收入的主要來源。
+              </li>
+              <li>
+                西方的卡提瓦半島有富足的金礦。
+              </li>
+            </ol>
+          </div>
+        </div>
+        <br />
+        <h2>斥候回報</h2>
+        <div class="hrn1"></div>
+        <div class="n1">
+          <ul>
+            <li>
+              法蘭西斯科‧德‧阿爾梅達到達了印度西北的內陸城鎮。在他為了他兒子洛倫索的死報仇雪恨之前，他不會讓出總督的位置。
+              阿方索‧阿爾布科克（紅色，葡萄牙人）帶著薩繆爾國王的一封任命信著陸了，他正信心滿滿地準備頂替法蘭西斯科成為新總督。
+              他出了名的專橫霸道，這使得他的人很容易被金錢和佈道所動搖。阿爾布科克遠征軍是由征服者，戟兵，強弩兵和火槍兵組成。
+              你要小心不要給葡萄牙同伴帶來沉重的傷亡，因為之後你會需要他們幫你一起對抗第烏。
+            </li>
+            <li>
+              第烏（紫色，波斯人）是在西方的卡提瓦島半島的南海岸外的一座島城，它將會是接下來的主戰場。
+              第烏城由馬利克阿雅姿統治，他是一位皈依了伊斯蘭教的俄國人，第烏人的主力軍是戰象，重裝駱駝和一些受穆斯林伊瑪目激勵而參戰的聖騎士。
+              由於威尼斯工匠的協助，馬利克阿雅姿在城內建造了火砲塔。
+            </li>
+            <li>
+              第烏人還得到奧斯曼蘇丹（綠色，土耳其人）大軍的援助。這是一支由土耳其皇家親兵，火砲和突厥重騎（全覆式重裝騎兵）組成的先鋒隊，
+              他們駐紮在蘇拉特城周圍，與葡萄牙人只隔著康坎河。
+            </li>
+            <li>
+              阿米爾侯賽因阿爾庫爾迪指揮埃及馬木留克奴隸兵人（青色，薩拉森人）。他的艦隊基地在卡姆巴特海灣的北面，第烏人的北面。
+              馬木留克奴隸兵軍隊包括奴隸騎兵，馬弓騎兵和攻城武器組成，還有一小支由義大利派遣過來的僱傭兵援軍。
+              威尼斯人還用貿易港灣大商站來獲取資源供馬木留克奴隸兵人招兵買馬。摧毀這些港灣大商站就會摧毀馬木留克奴隸兵人。
+            </li>
+            <li>
+              最終，古吉拉特商人（黃色，波斯人）在西方乾旱的和富有黃金的卡提瓦島半島有貿易市集。對於這些精明的商人來說，
+              只選擇一邊站隊是沒有利可圖的，因為他們急於和葡萄牙做交易。
+            </li>
+          </ul>
+          <h2>玩家攻略</h2>
+          <div class="hrn1"></div>
+          <div class="n1">
+            <ol>
+              <li>
+                。
+              </li>
+              <li>
+                。
+              </li>
+            </ol>
+          </div>
+          <h2>細節</h2>
+          <div class="hrn1"></div>
+          <div class="n1">
+            <ul>
+              <li>
+                目標要求你殺死所有三個敵人領導人（包括M馬利克·艾亞茲）。 然而，實際上，馬利克·艾亞茲在戰鬥中倖存下來並且只被抓住，
+                之後一直住在他的莊園，直到他去世。
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <table class="table-bordered" width="100%">
+          <tr>
+            <td class="unit" colspan="2" style="background-color:#375d96;">
+              <div class="unititle2">
+                <nuxt-link to="/batt/A_Sons_Blood">兒子的血仇</nuxt-link>
+              </div>
+            </td>
+          </tr>
+          <tr>
+            <td colspan="2">
+              <div class="unititle"><img src="~assets/A_Sons_Blood.pg" alt="" /></div>
```

Comment:
副檔名錯誤
~assets/A_Sons_Blood.pg
並且沒有此圖片 @bookthief108 
```suggestion
              <div class="unititle"><img src="~assets/A_Sons_Blood.png" alt="" /></div>
```